src/main/java/frc/robot/PressureSensor.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.*;
+
+public class PressureSensor {
+  AnalogInput input;
+  public PressureSensor(AnalogInput input) {
+    this.input = input;
+  }
+
+  private static final double VOLTAGE_AT_EMPTY = 0.50;
+  private static final double VOLTAGE_AT_FULL = 2.70;
+  private static final double MAX_PRESSURE = 120.0;
```

Comment:
This should be fine for now, but with the compressor we've been using for testing, we've been maxing out around `110` with the pressure switch.

Reply:
Fixed!